.github/workflows/docker.yml

```diff
@@ -67,3 +67,11 @@ jobs:
             && github.event_name != 'pull_request'
             && startsWith(github.ref, 'refs/heads/master')
           }}
+      - name: Trigger Helmchart PR
+        if: github.event_name == 'workflow_dispatch' && github.event.inputs.version != 'nightly'
+        with:
+          token: ${{ secrets.NETDATABOT_GITHUB_TOKEN }}
+          repo: netdata/helmchart
+          workflow: Agent Version PR
+          ref: refs/heads/master
+          inputs: {"agent_version": "${{ github.event.inputs.version }}"}
```

Comment:
Can you give a link to the docs?

I wonder how it works - I assume this step suppose to do an HTTP request using GitHub API, but there is no `uses`. And is it only `repo`? or should be `owner` and `repo`?

Reply:
Oh, nice catch, I forgot to add the `uses` line...

I’ll have a fix pushed shortly for that.

Reply:
Updated now to properly include the `uses` key.

For reference, the action being used here is: https://github.com/benc-uk/workflow-dispatch

Reply:
Thanks!